app/src/main/java/org/schabi/newpipe/views/CustomCollapsingToolbarLayout.java

```diff
@@ -0,0 +1,36 @@
+package org.schabi.newpipe.views;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.core.view.ViewCompat;
+import com.google.android.material.appbar.CollapsingToolbarLayout;
+
+public class CustomCollapsingToolbarLayout extends CollapsingToolbarLayout {
+    public CustomCollapsingToolbarLayout(@NonNull final Context context) {
+        super(context);
+        overrideListener();
+    }
+
+    public CustomCollapsingToolbarLayout(@NonNull final Context context,
+                                         @Nullable final AttributeSet attrs) {
+        super(context, attrs);
+        overrideListener();
+    }
+
+    public CustomCollapsingToolbarLayout(@NonNull final Context context,
+                                         @Nullable final AttributeSet attrs,
+                                         final int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+        overrideListener();
+    }
+
+    /**
+     * CollapsingToolbarLayout overrides our logic with fitsSystemWindows and ruins the layout.
+     * Override Google's method
```

Comment:
Could you better explain in which way this overrides a method? At first I interpreted it as a Java-language override, but it is not and you meant override in the sense of overwrite, so please change this to "Override Google's behaviour"

Reply:
Done